ambari-server/src/test/java/org/apache/ambari/server/topology/ClusterInstallWithoutStartTest.java

```diff
@@ -436,4 +434,9 @@ public void tearDown() {
   public void testProvisionCluster() throws Exception {
     topologyManager.provisionCluster(request, "{}");
   }
+
+  private static ServiceResponse service(String serviceName, long serviceId) {
```

Comment:
Maybe this method should be moved into a utility class, since it appears to be repeated in several test cases.